velox/exec/tests/HashJoinTest.cpp

```diff
@@ -60,16 +60,18 @@ class HashJoinTest : public HiveConnectorTestBase {
         BatchMaker::createBatch(rightType, rightSize, *pool_));
 
     CursorParameters params;
-    params.planNode =
-        PlanBuilder()
-            .values({leftBatch}, true)
-            .hashJoin(
-                makeKeyNames(keyTypes.size(), "t_"),
-                makeKeyNames(keyTypes.size(), "u_"),
-                PlanBuilder().values({rightBatch}, true).planNode(),
-                filter,
-                concat(leftType->names(), rightType->names()))
-            .planNode();
+    auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
```

Comment:
Move the planNodeIdGenerator to the class level so that it needn't be created at the tests level unless needed ? Same for other tests below.

Reply:
It is not clear to me why this would be better. Currently, tests that need a generator make one and use it.